### DISCUSSION TITLE
Fortigate GeoIP Field Mapping

### DISCUSSION THREAD

**Discussion by robertgryder** [Security-Onion-Solutions/securityonion#6767] [Unsupported Versions] [answered]
Hey all,

Started ingesting our Fortigate syslogs with the Filebeat Fortinet module and it seems to be working. However, the fields containing geoip data are not being mapped as geo_point fields, and thus cannot be used in visualizations. I'm guessing the default geoip.location field was not used since there are two locations for most logs. I have destination.geo.location.lat/lon and source.geo.location.lat/lon, which are both mapped as numbers instead of geo_point. 

What's the best way to remedy this? I guess I could use a runtime field but that seems like a bandaid fix.

**Comment by weslambert** [comment, marked as answer]
You could try a runtime field, however it would probably be best to do it through the Elasticsearch ingest node pipeline config and/or the index template. This has been brought up before, so we are aware of the issue, but haven't had a chance to push a complete fix just yet.

**Comment by RagnarSTS** [comment]
Confirm this issue is already being tracked here: 
https://github.com/Security-Onion-Solutions/securityonion/issues/3152
